_examples/blockchain.adoc

```diff
@@ -0,0 +1,107 @@
+= Blockchain Analysis Notebook
+:page-link: blockchain
+:page-weight: 101
+:page-labels: [Python, Jupyter, spark-notebook]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: The blockchain notebooks are examples of how to explore graph data using GraphX and GraphFrames on top of OpenShift using Spark. It uses the real Bitcoin blockchain data to create transaction graph for the analysis.
```

Comment:
s/The blockchain notebooks/These blockchain notebooks/

s/using Spark/using Apache Spark/

s/create transaction graph/create a transaction graph/